.pre-commit-hooks.yaml

```diff
@@ -147,3 +147,9 @@
     entry: trailing-whitespace-fixer
     language: python
     files: \.(asciidoc|adoc|coffee|cpp|css|c|ejs|erb|groovy|h|haml|hh|hpp|hxx|html|in|j2|jade|json|js|less|markdown|md|ml|mli|pp|py|rb|rs|R|scala|scss|sh|slim|tex|tmpl|ts|txt|yaml|yml)$
+-   id: no-commit-to-branch
+    name: Dont commit to branch
```

Comment:
nit: apostrophe in "Don't"? You might have to fool YAML by not using a bareword, e.g.

```yaml
name: "Don't commit to branch"
```